presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -141,6 +141,30 @@ public void shouldRunQueryFromFile()
         assertThat(trimLines(presto.readRemainingOutputLines())).containsAll(nationTableBatchLines);
     }
 
+    @Test(groups = CLI, timeOut = TIMEOUT)
+    public void shouldStopOnError()
+            throws IOException, InterruptedException
+    {
+        File temporayFile = File.createTempFile("test-sql", null);
+        temporayFile.deleteOnExit();
+        Files.write("select * from hive.default.nations;\nselect * from hive.default.nation;\n", temporayFile, UTF_8);
+
+        launchPrestoCliWithServerArgument("--file", temporayFile.getAbsolutePath());
+        assertThat(trimLines(presto.readRemainingOutputLines())).isEmpty();
+    }
+
```

Comment:
Please add tests for asserting on presto cli exit code for success/failure cases.